config.yaml

```diff
@@ -4,14 +4,12 @@
 
 # path to R
 rscript_bin: /usr/bin/Rscript
-# Servers configured for heat tree analysis with the metacoder R package. String with all server hostnames separated by space: "server1 server2 server3 ..."
-heattree_server: "hpc-bc15-07 hpc-bc15-12 hpc06 hpc-rc11"
+# Servers configured for heat tree analysis with the R and the metacoder R package. String with all server hostnames separated by space: "server1 server2 server3 ..."
+heattree_server: "hpc-rc11"
 # Path to conda.sh file. eg. /mnt/ngsnfs/tools/miniconda3/etc/profile.d/conda.sh
```

Comment:
I thought we wanted to get rid of the `config.yaml`?

Reply:
Yes, that's another issue which hasn't been done yet. For this, we need to fully automate as far as possible the heattrees and heatmaps steps, and list paths to all the adapters in the `nextflow.config`